### PR TITLE
DOCSP-26229: improve-runcommand

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-26229
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/DOCSP-26229-improve-runcmd/fundamentals/crud/run-command/

See #178 for comments that this PR addresses.

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/fundamentals/crud/run-command.txt [hunk 1]
```diff
@@ -15,46 +15,50 @@ Run a Command
...
+
+For a full list of database commands and corresponding parameters, see
+the :ref:`Additional Information section <addl-info-runcommand>`.
 
 .. note:: Read Preference
 
    ``RunCommand()`` and ``RunCommandCursor()`` do not obey the read
-   preference set for the database. You can set a read preference for a
-   database command by passing a ``RunCmdOptions`` type to either method:
+   preference you set on your ``Database`` object. You can set a read preference for a
```
Comment (mongoKart) [inline source/fundamentals/crud/run-command.txt:55]: Issue: If someone else has set the read preference in a different way, does ``RunCommand()`` obey that? As written, it sounds like it will ignore just the one that I set.

Suggestion: Clarify scope of what ``RunCommand()`` will respect.

File: source/fundamentals/crud/run-command.txt [hunk 2]
```diff
@@ -63,7 +67,10 @@ Information <addl-info-runcommand>`.
 Response
 --------
 
-Each method returns a document or a cursor containing documents with the
+Each method returns a result or a cursor that contains the response
```
Comment (mongoKart) [inline source/fundamentals/crud/run-command.txt:70]: Issue: What format is the result?

File: source/fundamentals/crud/run-command.txt [hunk 3]
```diff
@@ -63,7 +67,10 @@ Information <addl-info-runcommand>`.
...
+from the database after you run a command. Each database command
```
Comment (mongoKart) [inline source/fundamentals/crud/run-command.txt:71]: Issue: "after you run a command" is a little vague

Suggestion: I know we try to avoid passive, but I think it makes sense here: "...from the database after the command has completed." (or something similar)

File: source/fundamentals/crud/run-command.txt [hunk 4]
```diff
@@ -63,7 +67,10 @@ Information <addl-info-runcommand>`.
...
+requests different information, so the response content
```
Comment (mongoKart) [inline source/fundamentals/crud/run-command.txt:72]: I: Is every command just a query (requesting info from the DB) or are there other types too?

S: If the commands do very different things, maybe change to "Each database command performs a different function" or similar

File: source/fundamentals/crud/run-command.txt [hunk 5]
```diff
@@ -114,32 +125,46 @@ run the ``count`` command on the ``flowers`` collection of the
 Output
 ~~~~~~
 
-In the output, you should see the number of documents in
-the collection that match the query as the value of the ``n`` field, as
-well as the command execution information:
+In the output, you should see fields explaining the
+execution of the count operation, such as the winning plan and rejected
+plans. The output also contains the explain command execution information:
```
Comment (mongoKart) [inline source/fundamentals/crud/run-command.txt:130]: Issue: I could see this being a little hard to parse, especially for someone not familiar with explain commands. For example, do we expect the reader to know what a "winning plan" is in this context?

I also initially had trouble parsing the last sentence. Maybe change to "The output also contains information about the execution of the ``explain`` command."

**Review by mongoKart — CHANGES_REQUESTED** [review]
Just a couple small things

File: source/fundamentals/crud/run-command.txt (hunk 1 above)
Comment (rustagir) [reply source/fundamentals/crud/run-command.txt:55]: What this means is that when you instantiate a database object, you can set a read preference. But, the RunCommand method(s) will not obey that read preference.
I think the change I made improves clarity. LMK

File: source/fundamentals/crud/run-command.txt (hunk 4 above)
Comment (rustagir) [reply source/fundamentals/crud/run-command.txt:72]: [Some commands](https://www.mongodb.com/docs/manual/reference/command/nav-crud/) do make changes to the database/collection. I will change the language accordingly.

File: source/fundamentals/crud/run-command.txt (hunk 5 above)
Comment (rustagir) [reply source/fundamentals/crud/run-command.txt:130]: I added a clause explaining what a winning plan is.

**Review by mongoKart — APPROVED** [review]
LGTM
